modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -599,22 +599,26 @@
   (fn []
     (.getString rs i)))
 
+(defn- ^Connection rs->conn [^ResultSet rs]
```

Comment:
I checked out your branch and changed this to

```suggestion
(defn- rs->conn ^Connection [^ResultSet rs]
```

and ran Eastwood and it's not issuing any warnings. Maybe you were missing an `:import` or something when you ran it but it works fine for me

Reply:
thanks for checking. applied in 5dfe03915caae54382645814d4802686cc2df4a5